code/modules/research/circuitprinter.dm

```diff
@@ -118,8 +118,8 @@ using metal and glass, it uses glass and reagents (usually sulfuric acis).
 		busy = 1
 		use_power(max(1000, (3750*amount/10)))
 		var/stacktype = stack.type
-		stack.use(amount)
-		if(do_after(usr, 15, TRUE, 5, BUSY_ICON_FRIENDLY))
```

Comment:
s the removal of the icon intended here?

Reply:
Same as above.